Nukeeper.BitBucketLocal/BitBucketLocalPlatform.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using NuKeeper.Abstractions.CollaborationModels;
+using NuKeeper.Abstractions.CollaborationPlatform;
+using NuKeeper.Abstractions.Configuration;
+using NuKeeper.Abstractions.Logging;
+using NuKeeper.BitBucketLocal.Models;
+using Repository = NuKeeper.Abstractions.CollaborationModels.Repository;
+
+namespace NuKeeper.BitBucketLocal
+{
+    public class BitBucketLocalPlatform : ICollaborationPlatform
+    {
+        private readonly INuKeeperLogger _logger;
+        private AuthSettings _settings;
+        private BitbucketLocalRestClient _client;
+
+        public BitBucketLocalPlatform(INuKeeperLogger nuKeeperLogger)
+        {
+            _logger = nuKeeperLogger;
+        }
+
+        public void Initialise(AuthSettings settings)
+        {
+            _settings = settings;
+            var httpClient = new HttpClient
+            {
+                BaseAddress = settings.ApiBase
+            };
+            _client = new BitbucketLocalRestClient(httpClient, _logger, settings.Username, settings.Token);
+        }
+
+        public Task<User> GetCurrentUser()
+        {
+            return Task.FromResult(new User(_settings.Username, "", ""));
+        }
+
+        public async Task OpenPullRequest(ForkData target, PullRequestRequest request, IEnumerable<string> labels)
+        {
+            var repositories = await _client.GetGitRepositories(target.Owner).ConfigureAwait(false);
+            var targetRepository = repositories.FirstOrDefault(x => x.Name.Equals(target.Name, StringComparison.InvariantCultureIgnoreCase));
+
+            var pullReq = new PullRequest
+            {
+                Title = request.Title,
+                Description = request.Body,
+                FromRef = new Ref
+                {
+                    Id = request.Head
+                },
+                ToRef = new Ref
+                {
+                    Id = request.BaseRef
+                }
+            };
+
+            await _client.CreatePullRequest(pullReq, target.Owner, targetRepository.Name).ConfigureAwait(false);
+        }
+
+        public async Task<IReadOnlyList<Organization>> GetOrganizations()
+        {
+            var projects = await _client.GetProjects().ConfigureAwait(false);
```

Comment:
The approach adopted in this repo is to not use `.ConfigureAwait(false)` as this is a dotnet tool, and therefore will only ever be run in an environment with no sync context.

Reply:
Okay... I've removed the `.ConfigureAwait(false)`